rosbag2_cpp/include/rosbag2_cpp/clocks/player_clock.hpp

```diff
@@ -35,6 +36,72 @@ namespace rosbag2_cpp
 class PlayerClock
 {
 public:
+  class JumpHandler final
+  {
+public:
+    using SharedPtr = std::shared_ptr<JumpHandler>;
+    using pre_callback_t = std::function<void ()>;
+    using post_callback_t = std::function<void (const rcl_time_jump_t &)>;
+
+    JumpHandler() = delete;
+    JumpHandler(const JumpHandler &) = delete;
+    const JumpHandler & operator=(const JumpHandler &) = delete;
+
+    /// \brief Creates JumpHandler object with callbacks for jump operation.
```

Comment:
Ok will. Change.

Reply:
@emersonknapp BTW. I had a question in mind. What a rational to use `C` style Doxygen comments in modern `C++` code?
```
/**
 *
 */
```
As for me it looks the same odd as using `C` style cast in `C++`.

May be it would be better to reconsider default style for Doxygen comments?

Reply:
BTW. In `Apex.AI` we are using `C++` for doxygen comments. The are several reasons that:
1. It's compact.
2. It's visually looks more distinguishable from regular `C` style comment.
Consider example:
```cpp
  /// \brief remove jump callbacks from processing list.
  /// \param handler Shared pointer to the JumpHandler object returned from JumpHandler::create(..)
```
versus `C` style.
```cpp
  /**
   * \brief remove jump callbacks from processing list.
   * \param handler Shared pointer to the JumpHandler object returned from JumpHandler::create(..)
   */
```
`C` style has 2 extra lines for opening and closing comment.

Reply:
For what it is worth, we tend to use C-style everywhere; see https://github.com/ros2/rclcpp/blob/master/rclcpp/include/rclcpp/node.hpp (or any other header in rclcpp).  I'd rather that rosbag2 follow the style we use elsewhere.

While we *could* change to C++-style everywhere, it doesn't seem like a good use of time.  Both work fine.